validphys2/src/validphys/app.py

```diff
@@ -23,37 +23,37 @@
 
 
 providers = [
-             'validphys.results',
-             'validphys.commondata',
-             'validphys.pdfgrids',
-             'validphys.pdfplots',
-             'validphys.dataplots',
-             'validphys.fitdata',
-             'validphys.arclength',
-             'validphys.sumrules',
-             'validphys.reweighting',
-             'validphys.kinematics',
-             'validphys.correlations',
-             'validphys.chi2grids',
-             'validphys.eff_exponents',
-             'validphys.paramfits.dataops',
-             'validphys.paramfits.plots',
-             'validphys.theorycovariance.construction',
-             'validphys.theorycovariance.output',
-             'validphys.theorycovariance.tests',
-             'validphys.replica_selector',
-             'validphys.closuretest',
-             'validphys.mc_gen',
-             'validphys.theoryinfo',
-             'validphys.pseudodata',
-             'validphys.renametools',
-             'validphys.covmats',
-             'validphys.hyperoptplot',
-             'validphys.deltachi2',
-             'validphys.n3fit_data',
-             'validphys.mc2hessian',
-             'reportengine.report',
-            ]
+        'validphys.results',
+        'validphys.commondata',
+        'validphys.pdfgrids',
+        'validphys.pdfplots',
+        'validphys.dataplots',
+        'validphys.fitdata',
+        'validphys.arclength',
+        'validphys.sumrules',
+        'validphys.reweighting',
+        'validphys.kinematics',
+        'validphys.correlations',
+        'validphys.chi2grids',
+        'validphys.eff_exponents',
+        'validphys.paramfits.dataops',
+        'validphys.paramfits.plots',
+        'validphys.theorycovariance.construction',
+        'validphys.theorycovariance.output',
+        'validphys.theorycovariance.tests',
+        'validphys.replica_selector',
+        'validphys.closuretest',
+        'validphys.mc_gen',
+        'validphys.theoryinfo',
+        'validphys.pseudodata',
+        'validphys.renametools',
+        'validphys.covmats',
+        'validphys.hyperoptplot',
+        'validphys.deltachi2',
+        'validphys.n3fit_data',
+        'validphys.mc2hessian',
+        'reportengine.report',
+        ]
```

Comment:
This edit is both adding to the diff and still leaving the formatting in a bit of a peculiar state: either revert these altogether or run e.g.  black (or this thing that I haven't used https://pypi.org/project/darker/) to not have this square braked dangling (and use four spaces).

Same for the formatting changes below.

Reply:
I think the problem is I did use black

Reply:
For me black formats that list to

```python
providers = [
    'validphys.results',
    'validphys.commondata',
    'validphys.pdfgrids',
    'validphys.pdfplots',
    'validphys.dataplots',
    'validphys.fitdata',
    'validphys.arclength',
    'validphys.sumrules',
    'validphys.reweighting',
    'validphys.kinematics',
    'validphys.correlations',
    'validphys.chi2grids',
    'validphys.eff_exponents',
    'validphys.paramfits.dataops',
    'validphys.paramfits.plots',
    'validphys.theorycovariance.construction',
    'validphys.theorycovariance.output',
    'validphys.theorycovariance.tests',
    'validphys.replica_selector',
    'validphys.closuretest',
    'validphys.mc_gen',
    'validphys.theoryinfo',
    'validphys.pseudodata',
    'validphys.renametools',
    'validphys.covmats',
    'validphys.hyperoptplot',
    'validphys.deltachi2',
    'validphys.n3fit_data',
    'validphys.mc2hessian',
    'reportengine.report',
]
```

which I find okay.

Reply:
Funny. I wonder whether I have a funny default set or I opened it with the wrong program or smt.
Anyway, I'll revert it when I'm in front of the computer, I just wanted to remove those lines.